src/directives/formioWizard.js

```diff
@@ -445,6 +445,17 @@ module.exports = function() {
 
         // Move onto the previous page.
         $scope.prev = function() {
+          var errors = $scope.checkErrors();
+          if (errors) {
+            $scope.pageHasErrors[$scope.currentPage] = true;
+            if (!$scope.formioOptions.wizardFreeNavigation) {
```

Comment:
Not worried about it, but would like to know. Getting a new build ready and will include this.